lib/ood_core/job/adapters/lsf/helper.rb

```diff
@@ -92,6 +92,14 @@ def batch_submit_args(script, after: [], afterok: [], afternotok: [], afterany:
     args += ["-W", (script.wall_time / 60).to_i] unless script.wall_time.nil?
     args += ["-L", script.shell_path.to_s] unless script.shell_path.nil?
 
+    # Skip if nil
+    env = script.job_environment || {}
```

Comment:
This line  (`env = script.job_environment || {}`)is duplicate of line 116 below. I guess delete the one below?